source/encryption.rst

```diff
@@ -0,0 +1,132 @@
+.. _encryption:
+
+Data at rest encryption
```

Comment:
Capitalize in titles.  Should be Data-at-Rest Encryption

Data-at-Rest when written out is typically hyphenated.